crates/types/src/traits/network.rs

```diff
@@ -274,7 +274,7 @@ pub trait ConnectedNetwork<K: SignatureKey + 'static>: Clone + Send + Sync + 'st
     async fn broadcast_message(
         &self,
         message: Vec<u8>,
-        recipients: BTreeSet<K>,
+        topic: Topic,
         broadcast_delay: BroadcastDelay,
     ) -> Result<(), NetworkError>;
```

Comment:
Why did broadcast DA not also change?

Reply:
DA broadcast is a direct send to all recipients instead of an actual broadcast

Reply:
I think we should remove `da_broadcast_message` and conditionally on the topic have the DA specific logic when needed (as in Libp2p)